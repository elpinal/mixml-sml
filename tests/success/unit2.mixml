[unit
  link X = {module A = {module t = [type :Type]}} with
  {
    module R =
      link Z1 =
        link Y = {module u = [type :Type]} with
        {module f = [val :ext X.A.t -> ext Y.u]}
      seals
      link Z2 = {module u = [type int]} with
      {module f = [val fun (x : ext X.A.t) -> 7]}
  }
]
