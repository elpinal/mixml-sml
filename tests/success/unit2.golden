[forall (!2@A.t : Type). exist (!3 : Type). {A : {t : [= !2]}, R : {f : [!2 -> !3]+, u : [= !3]}}]+
